File: python/code program/truck.py
from kendaraan import Kendaraan

class Truck(Kendaraan):
    def __init__(self, brand="", model="", year=0, price=0.0, kapasitas=0, gerbong=False):
        super().__init__(brand, model, year, price)
        self.kapasitas = kapasitas
        self.gerbong = gerbong
    
    def set_kapasitas(self, kapasitas):
        self.kapasitas = kapasitas
    
    def set_gerbong(self, gerbong):
        self.gerbong = gerbong
    
    def get_kapasitas(self):
        return self.kapasitas
    
    def get_gerbong(self):
        return self.gerbong

File: java/code program/Truck.java
public class Truck extends Kendaraan {
    
    private int kapasitas;
    private boolean gerbong;

    public Truck(String brand, String model, int year, double price, int kapasitas, boolean gerbong) {
        super(brand, model, year, price);
        this.kapasitas = kapasitas;
        this.gerbong = gerbong;
    }

    public int getKapasitas() {
        return kapasitas;
    }

    public void setKapasitas(int kapasitas) {
        this.kapasitas = kapasitas;
    }

    public boolean isGerbong() {
        return gerbong;
    }

    public void setGerbong(boolean gerbong) {
        this.gerbong = gerbong;
    }
}
